msos/msos.csproj

```diff
@@ -26,6 +25,7 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
     <Prefer32Bit>true</Prefer32Bit>
     <UseVSHostingProcess>false</UseVSHostingProcess>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Threre is only one place that I need it to be an unsafe code, on _WAITCHAIN_NODE_INFO_LOCK_OBJECT structure declaration, since ObjectName member suppose to be a fixed size array:  fixed byte ObjectName[128 * 2];